### PR TITLE
Task/latest from gl

### PR DESCRIPTION
**Pull request by Ianrfisher**
### Change description ###
Bring inline with JCDE Gitlab so we can move development over to HMCTS Github

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -63,49 +63,85 @@ withPipeline(type, product, component) {
...
       allowMissing         : true,
       alwaysLinkToLastBuild: true,
       keepAll              : true,
       reportDir            : "build/reports/pmd",
       reportFiles          : "smokeTest.html",
-      reportName           : "PMD SmokeTest Report"
+      reportName           : "PMD Smoke Test Report"
+    ]
+
+    publishHTML target: [
```
Comment (Ben-Edwards-CGI) [inline Jenkinsfile_CNP:120]: Missing PMC Unit Test Report

File: LICENSE [hunk 2]
```diff
@@ -1,21 +1,9 @@
-MIT License
+The MIT License (MIT)
 
-Copyright (c) 2023 HM Courts & Tribunals Service
+Copyright (c) 2018 HMCTS (HM Courts & Tribunals Service)
```
Comment (Ben-Edwards-CGI) [inline LICENSE:3]: Should this bee 2023?

File: src/main/resources/application-test.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+spring:
+  datasource:
+    url: jdbc:postgresql://localhost:5433/scheduler_application
```
Comment (Ben-Edwards-CGI) [inline src/main/resources/application-test.yml:3]: This should allow overrides I.E
jdbc:postgresql://${DB_HOST:localhost}:${DB_PORT:5433}/${DB_NAME:scheduler_application}

File: Jenkinsfile_CNP (hunk 1 above)
Comment (Ianrfisher) [reply Jenkinsfile_CNP:129]: now added

File: LICENSE (hunk 2 above)
Comment (Ianrfisher) [reply LICENSE]: corrected

File: src/main/resources/application-test.yml (hunk 3 above)
Comment (Ianrfisher) [reply src/main/resources/application-test.yml]: done

**Review by Ben-Edwards-CGI — APPROVED** [review]

**Pull request merged**